tests/foreman/cli/test_setting.py

```diff
@@ -380,3 +382,62 @@ def test_negative_update_send_welcome_email(value):
     """
     with pytest.raises(CLIReturnCodeError):
         Settings.set({'name': 'send_welcome_email', 'value': value})
+
+
+@run_in_one_thread
+class BruteForceLogin(CLITestCase):
+    """automate brute force protection limit configurable function"""
+    def setUp(self):
+        super(BruteForceLogin, self).setUp()
+        self.host_value = Settings.list({
+            'search': 'name=failed_login_attempts_limit'})[0]['value']
+
+    def tearDown(self):
+        super(BruteForceLogin, self).tearDown()
+        # reset failed_login_attempts_limit value
+        sleep(301)
+        Settings.set({
+            u'name': u'failed_login_attempts_limit',
+            u'value': self.host_value})
+
+    @tier1
+    def test_positive_failed_login_attempts_limit(self):
+        """automate brute force protection limit configurable function
+
+         :id: f95407ed-451b-4387-ac9b-2959ae2f51ae
+
+         :steps:
+            1. Make sure login works.
+            2. Save current value and set it to some lower value:
+            3. Try to login with wrong password till failed_login_attempts_limit
+            4. Make sure login now does not work:
+            5. Wait timeout - 5 minutes + 1 second
+            6. Verify you can now login fine
+            7. Return the setting to previous value
+
+         :expectedresults: failed_login_attempts_limit works as expected
+
+         :caseautomation: automated
+         """
+        result = ssh.command(
+            'hammer -u {0} -p {1} user list'
```

Comment:
instead of doing user list, you may create a product here and at the end assert the user is able to list that product even after return code is 0.

Reply:
we are doing this so that we can check login works or not.
for creating product we will also require org.